src/System.Management.Automation/engine/Humanizer.cs

```diff
@@ -0,0 +1,132 @@
+/********************************************************************++
+Copyright (c) Microsoft Corporation.  All rights resulterved.
+--********************************************************************/
+
+using System;
+using System.Globalization;
+using System.Management.Automation;
+using System.Text;
+using System.Resources;
+using System.Reflection;
+
+namespace Microsoft.PowerShell
+{
+    /// <summary>
+    /// Contains ToHumanString CodeMethod implementations for TimeSpan
+    /// </summary>
+    public static partial class ToStringCodeMethods
+    {
+        /// <summary>
+        /// ToHumanString implementation for TimeSpan
+        /// </summary>
+        /// <param name="time">time is TimeSpan object</param>
+
+        public static string ToHumanString(TimeSpan time)
```

Comment:
The easiest way is to simply move here all the code and resources of the `Humanizer` project. If it is a valid solution then let's do it.